src/App.jsx

```diff
@@ -1,8 +1,15 @@
+import { Heading, VStack, Box } from "@chakra-ui/react";
 import { ReactQueryDevtools } from "react-query/devtools";
 import { QueryClient, QueryClientProvider } from "react-query";
-import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import {
+  Route,
+  Routes,
+  Navigate,
+  BrowserRouter as Router,
+} from "react-router-dom";
 
-import Home from "./pages/Home";
+import SubmitForm from "./pages/SubmitForm";
+import UsersList from "./pages/UsersList";
```

Comment:
```suggestion
import UsersList from "./pages/UsersList";
import SubmitForm from "./pages/SubmitForm";
```

to follow your standard import rules